web/packages/teleterm/src/ui/DocumentCluster/StatusInfo.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {
+  DatabaseServer,
+  ResourceStatus,
+  SharedResourceServer,
+  UnifiedResourceDefinition,
+  useResourceServersFetch,
+} from 'shared/components/UnifiedResources';
+import { UnhealthyStatusInfo } from 'shared/components/UnifiedResources/shared/StatusInfo';
+
+import { ResourcesResponse } from 'teleport/services/agents';
```

Comment:
We shouldn't import from `teleport`. For now we can avoid that by directly returning: 
```
{
     agents: servers,
     startKey: resp.nextKey,
}
```
instead of assigning it the variable.

Reply:
agh, thanks for the catch